### ISSUE TITLE
Feature request: Delimiters support

### ISSUE THREAD

**Issue by duongthienlee** [sqlpad/sql-limiter#19]
Hi, this is a great library. Though with database like mysql, having no support for adding delimiter is quite inconvenient. Creating triggers, stored procedures and functions become difficult.

**Comment by rickbergfalk** [comment]
This is something that should be able to be added. Out of curiosity, what is your use case using this for triggers and stored procedures and functions? Is it just to allow parsing of these queries?

**Comment by duongthienlee** [comment]
Yes, that's right, We want to parse it. We are making an editor similar to SQLPad but for only mysql .  We want to allow users to use the `DELIMITER` command to change the current terminator .i.e. `;` so that the following query can be treated as one statement:

    if (1 = 1) then select 1; else select 2; end if
    
or this
    
    create function my_function() returns int deterministic begin return 1; end
